src/components/categoricaldisplay/ResultsPanel.jsx

```diff
@@ -13,25 +13,41 @@ function getItemCount(category) {
     return count;
 }
 
-const CategoryTitleClassName = (categoryId, activeHeadKey, activeHeadTop) => {
+const CategoryTitleClassName = (categoryId, activeHeadKey, activeHeadTop, categorySubtitle) => {
```

Comment:
@nazaninreihani if `categorySubtitle` is a boolean variable, it'd be better to name in a way that conveys its value. e.g `hasCategorySubtitle`, `hasSubtitle`, ...